modal/sandbox.py

```diff
@@ -91,7 +91,38 @@ async def _get_logs():
         return data
 
 
+class _StreamWriter:
+    """Provides an interface to buffer and write logs to a sandbox stream (`stdin`)."""
+
+    def __init__(self, sandbox_id: str, client: _Client):
+        self._index = 1
+        self._sandbox_id = sandbox_id
+        self._client = client
+
+    def get_next_index(self):
+        index = self._index
+        self._index += 1
+        return index
+
+    async def write(self, input: str):
```

Comment:
Should this be `bytes` not `str`? Don't know why we'd force the client to write strings.